pandas/tests/frame/test_analytics.py

```diff
@@ -2237,13 +2237,20 @@ def test_clip_with_na_args(self):
         """Should process np.nan argument as None """
         # GH # 17276
         tm.assert_frame_equal(self.frame.clip(np.nan), self.frame)
-        tm.assert_frame_equal(self.frame.clip(upper=[1, 2, np.nan]),
-                              self.frame)
-        tm.assert_frame_equal(self.frame.clip(lower=[1, np.nan, 3]),
-                              self.frame)
         tm.assert_frame_equal(self.frame.clip(upper=np.nan, lower=np.nan),
                               self.frame)
 
+        # GH #19992
```

Comment:
can you comment under the commit number what this is testing

use 
```
result = 
expected = 
tm.assert_frame_equal(....)
```

do the tests in order for axis=0, axis=0, IOW so you don't have the df1, df2 variables

Reply:
Amended. Your format is better. Thanks.